src/ts/app.ts

```diff
@@ -0,0 +1,91 @@
+import Equation from "./equation.js";
+
+export default class App {
+  totalDiv: HTMLDivElement;
+  constructor() {
+    const equation = new Equation();
+    this.totalDiv = document.querySelector("#total") as HTMLDivElement;
+    document
+      .querySelector("div.modifiers.subgrid")!
```

Comment:
```suggestion
    $("div.modifiers.subgrid")!
```
https://github.com/transcendence42/javascript-calculator/pull/4/files#r645262877